ansible/roles/touchforms/templates/localsettings.py.j2

```diff
@@ -34,4 +34,4 @@ POSTGRES_DATABASE = {
 }
 ### END POSTGRES CONFIG ###
 
-FORMPLAYER_INTERNAL_AUTH_KEY={{ localsettings.FORMPLAYER_INTERNAL_AUTH_KEY }}
+FORMPLAYER_INTERNAL_AUTH_KEY="{{ localsettings.FORMPLAYER_INTERNAL_AUTH_KEY }}"
```

Comment:
Oops, sorry I missed this. Nit: ideally there'd be a single space around the `=`